Common/UI/Elements/UIImageItemSlot.cs

```diff
@@ -221,7 +221,7 @@ protected virtual void UpdateInteraction()
 			Main.HoverItem = Item.Clone();
 			Main.HoverItem.tooltipContext = Context;
 		}
-
+		
```

Comment:
Whitespace oopsie. Wouldn't have cared but this will conflict with some of my changes.